packages/analytics-core/test/timeline.test.ts

```diff
@@ -128,6 +128,62 @@ describe('timeline', () => {
       await timeline.deregister(before.name);
       expect(beforeExecute).toHaveBeenCalledTimes(0);
     });
+
+    test("should not pass event's extra to destination plugins", async () => {
```

Comment:
We want to pass the extra to the destination plugins to allow users to access it, but do not want to include it in the payload sent to the server in the Amplitude Destination.

Reply:
@kevinpagtakhan I noticed that the Amplitude Destination Plugin is named `Destination` do you think we could rename it to be more explicit e.g. `AmplitudeDestinationPlugin`?

Reply:
Sure, that makes sense!

Reply:
> We want to pass the extra to the destination plugins to allow users to access it

Fixed

>  noticed that the Amplitude Destination Plugin is named Destination do you think we could rename it to be more explicit e.g. AmplitudeDestinationPlugin?

In this PR?

Reply:
Thanks @falconandy we can rename the plugin in another PR. I just wanted to call it out.